_data/projects/SoCLI.yml

```diff
@@ -0,0 +1,11 @@
+name: SoCLI
+desc: Stack overflow command line client. Search and browse stack overflow without leaving the terminal.
+site: https://github.com/gautamkrishnar/socli
+tags:
+- stackoverflow
+- commandline
+- cli
+- python
+upforgrabs:
+  name: first-timers-only
+  link: https://github.com/gautamkrishnar/socli/labels/
```

Comment:
This should point to the list of issues: `https://github.com/gautamkrishnar/socli/labels/first-timers-only`